sharness/dependencies/ipfs-update/test-dist/testnew.go

```diff
@@ -167,8 +167,13 @@ func waitForApi(ipfspath string) error {
 }
 
 func TestBinary(bin, version string) error {
```

Comment:
wow, dealing with godeps is annoying... this file needs to be moved into `Godeps/_workspace/src/github.com/ipfs/ipfs-update/test-dist`